docs/versionPolicy.md

```diff
@@ -0,0 +1,145 @@
+# Versioning
+
+Running `kustomize` means one is running a
+particular version of a program, reading a
+particular version of a [kustomization] file.
+
+## Program versioning
+
+The command `kustomize version` prints a three field
+version tag that aspires to [semantic versioning].
+
+When enough changes have accumulated to
+(subjectively) warrant a new release, a [release process]
+is followed, and the fields in the version number
+are bumped per semver.
+
+## Kustomization type versioning
+
+At the time of writing, a [kustomization] file is just
```

Comment:
How about changing `At the time of writing` to `Until 2.0.0`.

Reply:
done!